src/routes/backingImage/DiskStateMapDetail.js

```diff
@@ -31,7 +31,7 @@ const modal = ({
     switch (event.key) {
       case 'delete':
         confirm({
-          title: `Are you sure you want to delete backing iamge in ${record.disk} ?`,
+          title: `Are you sure to delete the images from the disk ${record.disk} ?`,
```

Comment:
Singular `image` should be used here

Reply:
@smallteeths sorry, just merged. Please help fix this typo later on. Thanks.

Reply:
ok

Reply:
wu-shuo suggested changing `images` to `image file/files`